src/hooks/useConnections.ts

```diff
@@ -22,14 +22,23 @@ import { SourceTargetState, ChainDetails, ConnectionChainInformation, Configs }
 interface Props {
   connectionDetailsOne: ConnectionChainInformation;
   connectionDetailsTwo: ConnectionChainInformation;
+  initConnectionDetailsOne: ConnectionChainInformation;
+  initConnectionDetailsTwo: ConnectionChainInformation;
```

Comment:
What's wrong with `1` and `2` that we have to actually spell them? Also I'd very much prefer if we used `arrays` everywhere, at some point we might need to connect to `three` or even `dozen` of other chains and adding more configuration variables to this interface is not the right way to go.